samples/WebApp/README.md

```diff
@@ -0,0 +1,47 @@
+# Sample Web App
+
+This sample web application connects to your GitHub account using the Apps
+Script OAuth2 library and displays some information about the repositories you
+own. It demonstrates some best practices for using this library in a web app.
+
+## Setup
+
+You can easily setup your own copy of this sample using the
+[`clasp`](https://github.com/google/clasp) command line tool. Install it and
+run `clasp login` before executing the commands below.
+
+1.  Run the following commands in this directory:
+    ```sh
+    clasp create "Sample Web App"
+    clasp push
+    clasp open
+    ```
+1.  In the opened script editor, run the function `logRedirectUri`. Open the
+    logs (**View > Logs**) and copy the URL shown.
+1.  Open the [GitHub Developer console](https://github.com/settings/developers)
+    and create a new OAuth App. Paste the URL you copied above into the field
+    **Authorization callback URL**.
+1.  Edit the file `Code.gs` and uncomment the lines defining the variables
+    `CLIENT_ID` and `CLIENT_SECRET`. Paste the corresponding values from the
+    GitHub developer console into those variables.
+1.  Run the following commands:
+    ```sh
+    clasp version "Initial version"
+    clasp deploy 1 "Web App"
```

Comment:
You could just `clasp deploy` unless you really want the version description.

Reply:
Nice!